modules/core/src/main/java/org/apache/ignite/internal/processors/cache/GridCacheContextInfo.java

```diff
@@ -53,7 +55,25 @@
      * @param clientCache Client cache or not.
      */
     public GridCacheContextInfo(GridCacheContext<K, V> cctx, boolean clientCache) {
-        config = cctx.config();
+        config = new CacheConfiguration(cctx.config());
```

Comment:
Let's use `new CacheConfiguration<>` to remove compiler warning.

Reply:
This doesn't help, `cctx.config()` returns not parameterized CacheConfiguration. 
I suppose that we should either to do a massive refactoring, or leave it as is.